exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricConfiguration.java

```diff
@@ -215,6 +226,18 @@ public final Builder setProjectId(String projectId) {
     /** Sets the endpoint where to write Metrics. Defaults to monitoring.googleapis.com:443. */
     public abstract Builder setMetricServiceEndpoint(String endpoint);
 
+    /**
+     * Sets the {@link MetricConfiguration} to configure the exporter to write metrics via {@link
+     * com.google.cloud.monitoring.v3.MetricServiceClient#createServiceTimeSeries(String, List)}
+     * method. By default, this is false.
+     *
+     * @param useServiceTimeSeries a boolean indicating whether to use {@link
+     *     com.google.cloud.monitoring.v3.MetricServiceClient#createServiceTimeSeries(String, List)}
+     *     method for writing metrics to Google Cloud Monitoring.
+     * @return this
+     */
+    public abstract Builder setUseServiceTimeSeries(boolean useServiceTimeSeries);
```

Comment:
Not sure if this is a java pattern, but does this need an argument?  I.e. setUserServiceTimeSeries()?

Reply:
I think it is common to pass a  parameter when using `setFoo` methods.

I would think if you wish to avoid passing the parameter,  then the method should be called something like `enableFoo`. Consequently, you would then also need to add `disableFoo` - but in our current situation this would make the builder pattern inconsistent.

Reply:
I think the current approach is acceptable, then.